src/apolloServer.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { AppDataSource } from './data-source';
+import { ApolloServer } from 'apollo-server';
+import { ApolloServerPluginLandingPageLocalDefault } from 'apollo-server-core';
+import { User } from './entity/User';
+
+interface UserInput {
+  id?: number;
+  name: string;
+  email: string;
+  password: string;
+  birthdate: string;
+}
+
+async function initApolloServer() {
```

Comment:
```suggestion
export async function initApolloServer() {
```

Asism não precisa exportar lá embaixo.

Reply:
Modificado.